gulpfile.js

```diff
@@ -74,21 +74,6 @@ gulp.task('lint', function() {
     }))
 });
 
-gulp.task('functional-test', ['compile'], function() {
-  gulp.src('src/test/functional/functional-tests.js')
-  .pipe(sourcemaps.init())
-  .pipe(babel())
-  .pipe(sourcemaps.write('.'))
-  .pipe(gulp.dest('dist/test'))
-  .on('end', function() {
-    gulp.src('dist/test/functional-tests.js')
-    .pipe(mocha({
-      reporter: 'spec',
-      ui: 'bdd',
-    }))
-  })
-})
-
```

Comment:
@brendanashworth sometimes we might want to run just the functional tests, so you can keep this code.